tests/untl_structure_test.py

```diff
@@ -587,3 +587,43 @@ def test_generate_form_data(_):
     assert isinstance(fg, us.FormGenerator)
     # Check missing children were added.
     assert len(metadata.children) == len(metadata.contained_children)
+
+
+@pytest.mark.parametrize('test_input_content, test_output', [('True', True), ('False', False)])
+def test_Metadata_is_hidden(test_input_content, test_output):
+    """Check if a UNTL element is hidden."""
+    metadata = us.Metadata()
+    system = us.Meta(content='DC', qualifier='system')
+    hidden = us.Meta(content=test_input_content, qualifier='hidden')
+    metadata.children = [system, hidden]
+    assert metadata.is_hidden is test_output
+
+
+@pytest.mark.parametrize('test_input_content, test_output', [('True', False), ('False', True)])
+def test_Metadata_is_unhidden(test_input_content, test_output):
+    """Check if a UNTL element is unhidden."""
+    metadata = us.Metadata()
+    system = us.Meta(content='DC', qualifier='system')
+    hidden = us.Meta(content=test_input_content, qualifier='hidden')
+    metadata.children = [system, hidden]
+    assert metadata.is_unhidden is test_output
+
+
+def test_Metadata_make_hidden():
+    """Test if a UNTL unhidden element is altered to hidden."""
+    metadata = us.Metadata()
+    system = us.Meta(content='DC', qualifier='system')
+    unhidden = us.Meta(content='False', qualifier='hidden')
+    metadata.children = [system, unhidden]
+    metadata.make_hidden()
+    assert metadata.is_hidden is True
```

Comment:
Just a couple small things for this test and the next:
1. I would recommend parametrizing these two tests so that they are _both_ tested against hidden and unhidden records, just for completeness.
2. We check the hidden status at the end, but do you think it might be nice to also assert that the record is hidden/unhidden before we call make_hidden/make_unhidden so we can very clearly see when a change has occurred?